setup.py

```diff
@@ -26,51 +26,60 @@ def get_long_description():
     Replaces relative references to svg images to absolute https references.
     """
 
-    with open('README.md') as f:
+    with open("README.md") as f:
         read_me = f.read()
 
     def replace_relative_with_absolute(match):
         svg_path = match.group(0)[1:-1]
-        return ('(https://github.com/google/pybadges/raw/master/'
-                '%s?sanitize=true)' % svg_path)
+        return (
+            "(https://github.com/google/pybadges/raw/master/"
+            "%s?sanitize=true)" % svg_path
+        )
 
-    return re.sub(r'\(tests/golden-images/.*?\.svg\)',
-                  replace_relative_with_absolute,
-                  read_me)
+    return re.sub(
+        r"\(tests/golden-images/.*?\.svg\)",
+        replace_relative_with_absolute,
+        read_me,
+    )
 
 
 setup(
-    name='pybadges',
-    version='2.2.0',
-    author='Brian Quinlan',
-    author_email='brian@sweetapp.com',
+    name="pybadges",
+    version="2.2.0",
+    author="Brian Quinlan",
+    author_email="brian@sweetapp.com",
     classifiers=[
-        'Development Status :: 5 - Production/Stable',
-        'Intended Audience :: Developers',
-        'Intended Audience :: Developers',
-        'License :: OSI Approved :: Apache Software License',
-        'Programming Language :: Python',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: 3.7',
-        'Operating System :: OS Independent',
+        "Development Status :: 5 - Production/Stable",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Operating System :: OS Independent",
     ],
-    description='A library and command-line tool for generating Github-style ' +
-                'badges',
+    description="A library and command-line tool for generating Github-style "
+    + "badges",
     keywords="github gh-badges badge shield status",
-    package_data={'pybadges': ['badge-template-full.svg',
-                               'default-widths.json']},
+    package_data={
+        "pybadges": [
+            "badge-template-full.svg",
+            "default-widths.json",
+            "py.typed",
```

Comment:
I think that this is the only semantic change - could you limit your changes to that?

Reply:
Yes, sure. 

I have automatic linting using `black` on my editor. and I forgot to disable it. I'll limit my changes to that.